codalab/lib/codalab_manager.py

```diff
@@ -318,11 +318,11 @@ def session(self):
     @cached
     def default_user_info(self):
         info = self.config['server'].get(
-            'default_user_info', {'time_quota': '1y', 'disk_quota': '1t'}
+            'default_user_info', {'disk_quota': '1t', 'parallel_job_quota': '10'}
```

Comment:
make `10` an integer

Reply:
Also, 10 sounds like a lot given that we're likely to not have more than 10 runs at once.  Maybe 3?